### ISSUE TITLE
Nodemon and npm start should report what port is being used

### ISSUE THREAD

**Issue by jrhorn424** [ga-wdi-boston/express-template#23]

**Comment by gaand** [comment]
They do if you run it like this:

``` sh
DEBUG=express-template:server npm start
```

**Comment by raq929** [comment]
@gaand I tried that and it did not tell me the port. =/

**Comment by raq929** [comment]
NEVERMIND. (I had already updated to express-api-template. But had not changed your command.)

**Comment by gaand** [comment]
Add this (and related?) to the README:

```sh
DEBUG=express-api-template:server <npm start|nodemon>
```

**Comment by gaand** [comment]
Fixed in edde1a4072bafd22101ecafa78b417a54dcd3203

and 6e72ab528a202ef645832495592573b947666f69
